src/JuliusSweetland.OptiKey.UnitTests/Extensions/PointExtensionsTests.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections.Generic;
+using System.Windows;
+using JuliusSweetland.OptiKey.Extensions;
+using NUnit.Framework;
+
+namespace JuliusSweetland.OptiKey.UnitTests.Extensions
+{
+    [TestFixture]
+    public class PointExtensionsTests
+    {
+        [Test]
+        public void TestCalculateCentrePoint()
```

Comment:
When unit testing I noticed that PointExtensions.CalculateCentrePoint is using Convert.ToInt32  which rounds to an even number.

 According to https://msdn.microsoft.com/en-us/library/ffdk7eyz.aspx

"If value is halfway between two whole numbers, the even number is returned; that is, 4.5 is converted to 4, and 5.5 is converted to 6."

Not sure if this rounding is enough to throw anything off, but I figured I would mention it.